reco_utils/dataset/numpy_splitters.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+'''
+Collection of numpy based splitters
+
+'''
+
+import numpy as np
+import random
+import math
+from decimal import Decimal
+
+
+X = np.random.randint(low= 0, high=6, size= ( 20, 50) )
+
+def round(s):
```

Comment:
This function doesn't look very efficient, looping through an array. 

Tell me if what you wanted is this:

```
np.around(s).astype(np.int)
```

Reply:
yes, this does the job. I use a different convention for 0.5 (round to lower int) but it is just a convention and it does not matter much. I will change the code accordingly. Thanks!